app/models/article.rb

```diff
@@ -388,6 +388,16 @@ def self.unique_url_error
 
   scope :eager_load_serialized_data, -> { includes(:user, :organization, :tags) }
 
+  scope :above_average, lambda {
+    order(:score).where("score >= ?", average_score)
+  }
+
+  def self.average_score
+    return 0 unless count > 0 # rubocop:disable Style/NumericPredicate
```

Comment:
I'm maybe just not a fan of `Style/NumericPredicate` _anyway_, but it's _particularly_ important here that we're avoiding a divide-by-zero issue, so I think it's clearer without the numeric predicate in this case.

Reply:
I refactored the organization suggestion query to use ActiveRecord's `average` in 58fd17a882. (I spent a little more time trying to wrap my head around the [user suggestion query](https://github.com/forem/forem/blob/a345aabc2b60a281c7f7c47e63543e7aa2193004/app/services/users/suggest_recent.rb) and realized it might be helpful for refactoring it in the future if both suggesters can implement similar math. I also _really_ want to refactor the user suggester, but I think it should wait a separate PR.)